BeerOverflowWindowsApp/BarFileReader.cs

```diff
@@ -0,0 +1,23 @@
+﻿using BeerOverflowWindowsApp.DataModels;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BeerOverflowWindowsApp
+{
+    class BarFileReader : IReadable
+    {
+        private const string filePath = @".\barsData.txt";
+
+        public BarDataModel ReadData()
```

Comment:
Maybe it would be a good idea to make ReadData and SaveData methods static, then there would be no need to create objects of BarFileReader and BarFileWriter classes, since you never keep references to them anyways.

Reply:
It would be easier to call them, but If I make them static, I can't use the Interfaces. Not sure what to do :/

Reply:
Oh yeah, i didn't know you can't use static methods in interfaces. Well i think both solutions have their own benefits, maybe we implement it as you made it now, and we will change it later if we feel the need.